queries.sql

```diff
@@ -87,3 +87,50 @@ Where date_of_birth Between '1990-1-1' and '2000-1-1'
 Group by species;
 
 Select * from animals;
+
+
+---------------------------Join------------------------------
+
+Select * from animals;
+Select * from owners;
+Select * from species;
+
+
+Select o.full_name as owner, a.name as animal_name
+From owners o
+Join animals a On o.id = a.owner_id
+Where o.full_name = 'Melody Pond';
+
+Select a.name, s.name
+From species s
+Join animals a On s.id = a.species_id
+Where s.name = 'Pokemon';
+
+Select o.full_name as owner, a.name as their_pet
+From owners o
+Left Join animals a On o.id = a.owner_id;
```

Comment:
- [ ] Also, this query `List all owners and their animals, remember to include those that don't own any animal.` needs to return 11 rows in total. But yours is currently returning only 8 so kindly fix that as well :+1:

Reply:
> * [ ]  Also, this query `List all owners and their animals, remember to include those that don't own any animal.` needs to return 11 rows in total. But yours is currently returning only 8 so kindly fix that as well 👍

This is the result of the query. Can you show me other way???
![Screenshot 2023-11-04 182308](https://github.com/Pi1998/Vet-Clinic/assets/110252746/5eb9fe10-5bd1-41d7-905d-75742941a61e)